### ISSUE TITLE
edgecos model used with Edge-Core FE L2 Switch ES3528M triggers falseopositive diffs due to uptime and memory utilization changes

### ISSUE THREAD

**Issue by sq9mev** [ytti/oxidized#1364]
We're getting diffs like this every scan:

```
diff --git a/192.168.127.150 b/192.168.127.150
index cf6e28e..8e3a2f0 100644
--- a/192.168.127.150
@@ -378,8 +378,8 @@ end
 !   Free Policy Control Entries  : 760
 !   TCAM Utilization             : 25.78%
 !  ------ ---------- --- ---------- -------------- --------------
-!  Free     10010916  43        130          77007        9896472
-!  Alloc    12814268  57      58109            220              -
 ! 
 !  Alarm Configuration
 !   Rising Threshold         : 90%
@@ -387,7 +387,7 @@ end
 ! 
 ! System OID String: 1.3.6.1.4.1.259.6.10.94
 ! System Information
-!  System Up Time:          49 days, 20 hours, 57 minutes, and 4.47 seconds
 !  System Name:             ***
 !  System Location:         ***
 !  System Contact:          ***
```

Well, i think we have 3 problems here:

1. https://github.com/ytti/oxidized/blob/master/lib/oxidized/model/edgecos.rb#L16 `show access-list tcam-utilization`
1. https://github.com/ytti/oxidized/blob/master/lib/oxidized/model/edgecos.rb#L20 `show memory`
1. https://github.com/ytti/oxidized/blob/master/lib/oxidized/model/edgecos.rb#L24 `show system`

As of two first problems, i think that oxidized as "configuration backup software" is not "network monitoring softwqare", and these statements could be safely removed regardless of device model.

The last one… well, uptime could be filtered.

What do you think?

**Comment by wk** [comment]
Closed by #1366

**Comment by wk** [comment]
Looks reasonable. 

Oxidized typically aims to collect more than just configuration (in order to be able to determine the context / equipment to which the configuration applied), but there is indeed an implicit goal to avoid overly volatile output.
